lib/msal-electron-proof-of-concept/samples/msal-electron-sample-app/src/Main.ts

```diff
@@ -53,19 +53,20 @@ export default class Main {
             clientId: '5b5a6ef2-d06c-4fdf-b986-805178ea4d2f',
         };
         this.msalApp = new PublicClientApplication(msalAuthConfig);
-        console.dir(this.msalApp);
     }
 
     // Sets a listener for the AcquireToken event that when triggered
     // performs the authorization code grant flow
-    private static listenForAcquireToken(): void {
+    private static async listenForAcquireToken(): Promise<void> {
```

Comment:
why 'async' when using 'promise'?

Reply:
Yeah, fixing that in #863, this PR is still a WIP. If you want to take a look at that one, I'd appreciate it!